### DISCUSSION TITLE
itwin.js built-in rpc performance issues

### DISCUSSION THREAD

**Discussion by shenshengyi** [iTwin/itwinjs-core#3909] [Q&A]
hi, i need to send 1500 elmentId from backend to frontend,
In the same network environment,
Wrote a custom to RPC RpcInterface, but it takes nearly 1 minute.
Then I use restful api, the front end uses axios to send http request, and the query request improves a lot.it only takes 100ms.
I'm wondering what could be the reason for such a big gap?
Or what factors may need to be paid attention to when writing a custom RPC RpcInterface? thanks.

**Comment by aruniverse** [comment]
Please provide code for your example

**Reply by ayushibasaiwal** [reply]
@shenshengyi  Can you please provide a code so that we can understand it better?
